blocks/adp-share-header/adp-share-header.css

```diff
@@ -0,0 +1,67 @@
+.section.adp-share-header-left-container {
```

Comment:
@TyroneAEM could you review again if all style classes added in this file are needed for the share-header block.

Reply:
I have reviewed the styles and removed the redundant styles that were copied over from the adp-collections-header that are not used